### PR TITLE
xorg-libXau: update to 1.0.9

### PR DESCRIPTION
**Pull request by dgilman**
###### Type(s)
- [x] bugfix

###### Tested on
macOS 10.13.6 17G5019
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: x11/xorg-libXau/Portfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PortSystem 1.0
 
 name		xorg-libXau
-version		1.0.8
+version		1.0.9
 revision        1
```
Comment (mf2k) [inline x11/xorg-libXau/Portfile]: The `revision` line needs to be reset to 0 when upgrading the `version`.

**Comment by pmetzger** [comment]
Do we need to wait the full three days?

**Comment by mf2k** [comment]
Given that it is openmaintainer and Jeremy has not been active lately, I think not.

**Comment by pmetzger** [comment]
@mf2k If it wouldn't be unspeakably rude, I'd suggest that we file Port Abandoned tickets for the X packages at least so that other people can take over maintaining them. :(

**Comment by mf2k** [comment]
It is certainly not rude. However, Jeremy is the domain expert for X11 on macOS, so I like having him be maintainer of these ports. He has left them open and I am pretty sure he has always been fine with other people keeping them updated for these relatively minor version increases.

**Comment by kencu** [comment]
Losing Jeremy would be a huge blow to MacPorts. Clearly he's busy as h*ll at Apple these days. We do modest changes and updates to his ports, but mostly everything is the way he set it up and we're coasting on that work.

**Comment by pmetzger** [comment]
@kencu It's been well over a year since he he's commented than a trivial amount. I think we've already lost him, de facto. I don't like it as he's kind of irreplaceable but it seems to be true.

**Comment by kencu** [comment]
I know he's been busy. 

There are so few people around with Jeremy's insight into the whole machine and his fabulous temperament, though. Imagine if you could what someone with Jeremy's skill set would go for, on an hourly basis... we should all take up a collection and send him to Napa for a weekend. Really.

**Comment by pmetzger** [comment]
Even if we do that, we still have to live with the fact that we are not likely to have much of his time any month soon. Even reality is unpleasant, it remains unavoidable.

**Pull request merged**